gpMgmt/bin/gpexpand

```diff
@@ -2110,7 +2111,17 @@ class ExpandTable():
         # check is atomic in python
         if not cancel_flag:
             if sql:
-                dbconn.execSQL(table_conn, sql)
+                try:
+                    dbconn.execSQL(table_conn, sql)
+                except DatabaseError as e:
+                    if re.search('DETAIL:\s*table has already been expanded', str(e)):
+                        logger.info('Table %s.%s seems to be already expanded, marking as done' % (self.dbname, self.fq_name))
+                    elif re.search('\s*could not open relation with (OID \d+ )?', str(e)) or re.search('\s*relation not found (OID \d+ )?', str(e)):
```

Comment:
I think, the change
```go
 or re.search('\s*relation not found (OID \d+ )?', str(e))
```
should be described in commit message

Reply:
@dnskvlnk , explain please why we need two conditions. The PR author believes that one condition is enough for 6X_STABLE https://github.com/greenplum-db/gpdb/pull/15892/files

Reply:
This was a message from 7x, so I removed it